changelog/template.rst

```diff
@@ -0,0 +1,19 @@
+{% for section, _ in sections.items() %} {% set underline = underlines[0] %}{% if section %}{{section}} {{ underline * section|length }}{% set underline = underlines[1] %}
```

Comment:
have you modified this? if not you shouldn't need it, I only added it to upstream to test it as astropy customised it.

Reply:
Yeah, I just copied it over - my bad. I was trying to fix https://github.com/PlasmaPy/PlasmaPy/pull/1052/checks?check_run_id=2012587897#step:6:643 this test failure, where the documentation is built within this tox env:

https://github.com/PlasmaPy/PlasmaPy/blob/6a161f09530ee3d6870ea00a5c95f5ef278e12fb/tox.ini#L39-L44

For some reason, it seems to be running into

https://github.com/OpenAstronomy/sphinx-changelog/blob/90e565cb35e3f80bdebd6da358ecfdc9d2ac9b14/sphinx_changelog/towncrier.py#L33

with `config = None`, which implies that this part

https://github.com/twisted/towncrier/blob/7f209109513534d4c3e9defc732fa43ce0ff99aa/src/towncrier/_settings.py#L60-L65

cannot locate pyproject.toml: https://github.com/PlasmaPy/PlasmaPy/blob/6a161f09530ee3d6870ea00a5c95f5ef278e12fb/pyproject.toml#L5-L9

which might warrant a `if config is None or config["template"] is None` in sphinx-changelog and definitely warrants more debugging on our end to figure out what happens to pyproject.toml :sweat_smile: